src/model/transaction/moveBlockBeforeInContentState.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule moveBlockBeforeInContentState
```

Comment:
Looks like a lot of similarity in code between this module and the one above. Can they be combined?